lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb

```diff
@@ -49,11 +49,11 @@ def self.run(params)
         [next_beta_version, next_alpha_version]
       end
 
-      def self.get_user_build_version(version, message)
+      def self.get_user_build_version(product_name:, version:, message:)
         UI.user_error!("[#{app}] Release branch for version #{version} doesn't exist. Abort.") unless Fastlane::Helper::GitHelper.checkout_and_pull(release: version)
-        release_version = Fastlane::Helper::Android::VersionHelper.get_release_version
+        release_version = Fastlane::Helper::Android::VersionHelper.get_release_version(product_name: product_name)
         message << "#{app}] Looking at branch release/#{version} as requested by user. Detected version: #{release_version[Fastlane::Helper::Android::VersionHelper::VERSION_NAME]}.\n"
-        alpha_release_version = Fastlane::Helper::Android::VersionHelper.get_alpha_version
+        alpha_release_version = Fastlane::Helper::Android::VersionHelper.get_alpha_version(product_name)
```

Comment:
Side note: we need to harmonize the parameter names for all `get_*_version` helper methods at some point, so that we're consistent with using `product_name:` named parameter for all of those, instead of having a mix like here. But that's something for another day, as the goal of this PR is mostly to make things work, limiting any refactoring to a minimum for now